### DISCUSSION TITLE
Not connecting to SFTP server one one of four identical installations??

### DISCUSSION THREAD

**Discussion by ScottKnick** [iterate-ch/cyberduck#14494] [Q&A]
We use Cyberduck to connect to a cloud server via SFTP. Because our use is relatively infrequent, we share a single login credential. On three of the four installations I've done (all on Macbooks) the program works fine, but on one the connection consistently fails with the message: "Exhausted available authentication methods."  I have checked the Cyberduck configuration on the failing Mac carefully, and have reinstalled the app a couple of times, with no improvement. Any ideas on what's happening?

**Comment by dkocher** [comment]
Check if the account has some custom settings in `~/.ssh/config` that might interfere. Supported configuration directives that influence the authentication attempts are [documented](https://docs.cyberduck.io/protocols/sftp/#openssh-configuration-interoperability).
